runtests.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+# Simple (too simple!) test running script, to tide us over until we get tox
+# etc. working properly.
+#
+# 1) Creates a venv named like 'test-venv-py36', and tries to install the
+#    required stuff into it. If you delete this it'll be recreated.
```

Comment:
nitpick for later: It would be better to use a path that starts with a dot like tox does, because tools like ag and rg (but not ack) won't search there, which is what we want.

Reply:
Hmm, maybe; I guess I was reluctant to make it invisible given that it's a weird ad hoc thing with lousy tooling, so people will probably end up needing to know about it :-).

This did at least remind me to add it to .gitignore though (I just pushed this directly to bleach-spike), which I know at least rg will ignore (and I think the others as well).